Packages/com.unity.inputsystem/InputSystem/Plugins/XR/TrackedPoseDriver.cs

```diff
@@ -186,10 +290,24 @@ void BindRotation()
             }
         }
 
-        void UnbindActions()
+        void BindTrackingState()
         {
-            UnbindPosition();
-            UnbindRotation();
+            if (m_TrackingStateBound)
+                return;
+
+            var action = m_TrackingStateInput.action;
+            if (action == null)
+                return;
```

Comment:
There may be a scenario where a user externally sets `trackingStateInput = null`
```suggestion
            if (m_TrackingStateInput == null)
                return;

            var action = m_TrackingStateInput.action;
            if (action == null)
                return;
```